docs/source/top_level_elements/language.rst

```diff
@@ -0,0 +1,172 @@
+language
+========
+
+About
+_____
+This section describes all the different types of language elements that we have in our Islandora repository right now.
+
+language has languageTerm subelement with an @authority of "iso639-2b" and a @type of "text"
+--------------------------------------------------------------------------------------------
```

Comment:
Headings here look weird.  Is the intention of headings like this to be a subheading of about or a sibling heading?

Reply:
Addressed hierarchy issues in 6d1fdfe, making subsubsections for my distinct cases